README.md

```diff
@@ -50,4 +50,16 @@ This repository is using `Reactjs` and `Redux` and following their naming conven
 ## Migrate media data
 ```
 bundle exec rake master_data:media_post
-```
\ No newline at end of file
+```
+
+## Deploy to develop server
+```
+git tag -f deploy_server_develop_env_<insert env here>
+git push -f <origin/upstream> deploy_server_develop_env_<insert env here>
```

Comment:
I think it is safe to just deploy all available environments of develop on new commit?
@vietthang207 please consult @andrewsource.

Reply:
It is safe, it just take 1-2 mins longer to build.

Reply:
Then please change the PR to only read tag and deploy for staging.

Reply:
Sometimes we need to use a different build env in dev (i.e. mainnet/ropsten)